packages/core/admin/admin/src/content-manager/pages/App/LeftMenu/index.js

```diff
@@ -61,8 +60,7 @@ const LeftMenu = () => {
       id: 'singleTypes',
       title: {
         id: getTrad('components.LeftMenu.single-types'),
-        defaultMessage:
-          '{number, plural, =0 {Single Types} one {Single Type } other {Single Types}}',
+        defaultMessage: 'Single Types',
         values: { number: intlSingleTypeLinks.length },
```

Comment:
Same here I think you don't use the number variable anymore!

Reply:
Very true, thanks! Will check and clean all of them :+1: